### ISSUE TITLE
HttpClientMetricsExtensions missing in prometheus-net 

### ISSUE THREAD

**Issue by NicolaiPetri** [prometheus-net/prometheus-net#314]
Are there any reason for not exposing HttpClientMetricsExtensions in prometheus-net version ? When doing console applications with hosted services and httpclients, it makes good sense to be able to track metrics for httpclients.

**Comment by sandersaares** [comment]
Good point! I will make the change. A prerelease build of the package should be available soon - please let me know if it seems to work for you.
